community/kernel/src/main/java/org/neo4j/kernel/impl/api/LockingStatementOperations.java

```diff
@@ -266,6 +257,21 @@ public void visit( long relId, int type, long startNode, long endNode )
         entityWriteDelegate.relationshipDelete( state, relationshipId );
     }
 
+    private void lockRelationshipNodes( KernelStatement state, long startNodeId, long endNodeId )
+    {
+        // Order the locks to lower the risk of deadlocks with other threads adding rels concurrently
```

Comment:
This comment is outdated because method is used for both relationship creation and deletion.